ga_resources/tasks.py

```diff
@@ -7,7 +7,7 @@
 from django.utils.timezone import utc
 
 
-@task(ignore_result=True)
+#@task(ignore_result=True)
 def refresh_resource(pk):
```

Comment:
Is this module still needed at all?  Can this functionality be folded into hs_core.tasks?